src/main/kotlin/no/nav/familie/ef/søknad/integration/SøknadClientUtil.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.søknad.integration
+
+import no.nav.familie.kontrakter.ef.ettersending.SøknadMedDokumentasjonsbehovDto
+import java.time.LocalDate
+
+object SøknadClientUtil {
+
+    fun filtrerVekkEldreDokumentasjonsbehov(søknader: List<SøknadMedDokumentasjonsbehovDto>): List<SøknadMedDokumentasjonsbehovDto> {
+        val fireUker: Long = 4 * 7; /* dager */
+        return søknader.filter { LocalDate.now().toEpochDay() - it.søknadDato.toEpochDay() < fireUker }
```

Comment:
Er `...{ it.søknadsdato.plusWeeks(4) > LocalDate.now() }` lettere å forstå?

Reply:
Fixed: 74b5ea4